modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ObjectBasedParameterValueResolver.java

```diff
@@ -67,4 +69,9 @@ public Object getParameterValue(String parameterName) throws ValueResolvingExcep
     }
     throw new ValueResolvingException("Unable to resolve value for the parameter: " + parameterName);
   }
+
+  @Override
+  public Map<String, ValueResolver<? extends Object>> getParameters() {
+    return new HashMap<>();
```

Comment:
Unsure what to return here, as this class has no access to the resolvers.

Reply:
this must be consistent with the `getParameterValue(String` method already present. Return a map made of the object's fields and values. You can use `StaticValueResolver` for the values

Reply:
fixed